Provide more details on how to generate the pages locally and on github pages when having additional dependencies (#3)
w3c/w3c-jekyll-theme: README.md

- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
- [Bundler](https://bundler.io/)

On Ubuntu (24.04 LTS):
```shell
sudo apt install ruby-dev ruby-bundler
# Set the path to your bundler data
mkdir ~/.bundler
bundle config path ~/.bundler
```

> On MacOS:
> ```shell
> brew install chruby ruby-install
> ruby-install ruby 3.2.3
> # Update your bash profile to use this version of Ruby
> echo "source $(brew --prefix)/opt/chruby/share/chruby/chruby.sh" >> ~/.bash_profile
> echo "source $(brew --prefix)/opt/chruby/share/chruby/auto.sh" >> ~/.bash_profile
> echo "chruby ruby-3.2.3" >> ~/.bash_profile
> 
> ## Confirm your Ruby and Bundler versions
> $ ruby --version
> ruby 3.2.3
> $ bundle version
> Bundler version 2.4.20
> ```

Reviewer: Split in 2 separate code blocks as it is for both Ubuntu and Mac platforms

```suggestion
On MacOS:
```shell
brew install chruby ruby-install
ruby-install ruby 3.2.3
# Update your bash profile to use this version of Ruby
echo "source $(brew --prefix)/opt/chruby/share/chruby/chruby.sh" >> ~/.bash_profile
echo "source $(brew --prefix)/opt/chruby/share/chruby/auto.sh" >> ~/.bash_profile
echo "chruby ruby-3.2.3" >> ~/.bash_profile
```

```shell
## Confirm your Ruby and Bundler versions
$ ruby --version
ruby 3.2.3
$ bundle version
Bundler version 2.4.20
```
```

### Theme Installation

> 1. **Specify your Jekyll dependencies (optional)**
>     Create the file `Gemfile` at the root of your project. You may skip this step if you don't need the Jekyll plugins listed below but you will have to specify the `jekyll-remote-theme` plugin in your `_config.yml` file (see the [Jekyll configuration](#jekyll-configuration) section):

Reviewer: Add missing new line

```suggestion
1. **Specify your Jekyll dependencies (optional)**

    Create the file `Gemfile` at the root of your project. You may skip this step if you don't need the Jekyll plugins listed below but you will have to specify the `jekyll-remote-theme` plugin in your `_config.yml` file (see the [Jekyll configuration](#jekyll-configuration) section):
```

### GitHub Pages

> There are two ways to deploy on GitHub pages depending on whether your site depends on specificy jekyll dependencies. By default, GitHub pages come with a bunch of [jekyll plugins](https://pages.github.com/versions/). If you need other dependencies, e.g. jekyll-toc or jekyll-commonmark,to be installed with bundler, you will have to use a GitHub action to generate and deploy the pages.

Reviewer: Fix typo and small wording improvements

```suggestion
There are two ways to deploy on GitHub pages depending on whether your site rely on specific Jekyll dependencies. By default, GitHub pages come with a bunch of [Jekyll plugins](https://pages.github.com/versions/). If you need other dependencies to be installed with bundler (e.g. `jekyll-toc` or `jekyll-commonmark`) you will have to use a GitHub action to generate and deploy the pages.
```
